ctgan/data_sampler.py

```diff
@@ -146,7 +146,7 @@ def dim_cond_vec(self):
 
     def generate_cond_from_condition_column_info(self, condition_info, batch):
         vec = np.zeros((batch, self._n_categories), dtype='float32')
-        id = self._discrete_column_matrix_st[condition_info["discrete_column_id"]
-                                             ] + condition_info["value_id"]
+        id = self._discrete_column_matrix_st[condition_info["discrete_column_id"]] \
```

Comment:
This line is a bit wird altogether, combining two levels of brackets with an addition.
I think it would cleaner if we separate in two steps:

```python3
id = self._discrete_column_matrix_st[condition_info["discrete_column_id"]]
id += condition_info["value_id"]
```